docs/user-guide/configuring-at-tls-for-zowe-server.md

```diff
@@ -330,14 +440,15 @@ This scenario includes the following services:
 * Services that require an Open ID Connect (OIDC) token
 * Forwarded X.509 certificates
 
-In this case, it is necessary to have an Outbound rule from the service to the API Gateway.
+In case the service is running on z/OS we strongly recommend:
```

Comment:
I think the article should be all about z/OS, in any case there should be a note about cases where they are setting up AT-TLS to interface with a non-zOS (as the exception)